source/features/esc-to-cancel.tsx

```diff
@@ -0,0 +1,36 @@
+import select from 'select-dom';
+import onetime from 'onetime';
+import delegate from 'delegate-it';
+
+import features from '.';
+import onIssueTitleKeydown from '../github-events/on-issue-title-field-keydown';
+
+function eventHandler(event: delegate.Event<KeyboardEvent, HTMLTextAreaElement>): void {
+	const field = event.delegateTarget;
+	if (event.key === 'Escape') {
+		const cancelButton = select<HTMLLinkElement>(`
+				.js-cancel-issue-edit
+			`, field.form!);
+		// Cancel if there is a button, else blur the field
+		if (cancelButton) {
```

Comment:
For example `cancel` should **always** exist here, so no need for this condition